packages/utils/regl-renderer/src/rendering/commands/drawMesh/index.js

```diff
@@ -9,14 +9,14 @@ const drawMesh = (regl, params = { extras: {} }) => {
     geometry: undefined,
     color: meshColor
   }
-  const { geometry, dynamicCulling, useVertexColors, color, transparent } = Object.assign({}, defaults, params)
-
+  const { geometry, dynamicCulling, useVertexColors, color, visuals } = Object.assign({}, defaults, params)
```

Comment:
hmm... it seems that we never had a 'default' value for transparent. let's assign a proper 'default' for visuals. maybe {}?